### PR TITLE
Create a Caddy egg

### LINKED ISSUE #16: Create a Caddy egg
Currently there isn't really a egg that supports webhosting. A caddy egg would be the best solution.

Application: caddy-server
Homepage: https://caddyserver.com/

### PR DESCRIPTION
**Pull request by pascal-zarrad**
## Description
Create an egg that provides a Caddy webserver

## Related issue
Resolves #16

### CODE REVIEW

File: eggs/caddy/README.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# caddy
+Fast, multi-platform web server with automatic HTTPS
+
+### Installation
+Just import the .json in the **Nests** section of Pterodactyl.
+
+The Dockerfile is available here: [DockerHub](https://hub.docker.com/r/lazybytez/eggs/tags?page=1&ordering=last_updated&name=caddy-server)
```
Comment (elias-knodel) [inline eggs/caddy/README.md:7]: Does this even have a dockerfile?

Comment (pascal-zarrad) [reply eggs/caddy/README.md:7]: We're using the official `quay.io/parkervcp/pterodactyl-images:base_debian` image as a base.

**Review by elias-knodel — DISMISSED** [review]
LGTM :+1:

**Review by elias-knodel — APPROVED** [review]
LGTM 💯

**Pull request merged**